src/cltk/dependency/stanza_wrapper.py

```diff
@@ -242,18 +242,18 @@ def _load_pipeline(self):
         >>> isinstance(nlp_obj, stanza.pipeline.core.Pipeline)
         True
         """
-        models_dir = os.path.expanduser(
+        models_dir: str = os.path.expanduser(
             os.path.join("~", "stanza_resources")
         )  # TODO: Mv this a self. var or maybe even global
-        processors = "tokenize,mwt,pos,lemma,depparse"
-        lemma_use_identity = False
+        processors: str = "tokenize,mwt,pos,lemma,depparse"
+        lemma_use_identity: bool = False
         if self.language == "fro":
             processors = "tokenize,pos,lemma,depparse"
             lemma_use_identity = True
         if self.language in ["chu", "got", "grc", "lzh"]:
             # Note: MWT not available for several languages
             processors = "tokenize,pos,lemma,depparse"
-        nlp = stanza.Pipeline(
+        nlp: stanza.Pipeline = stanza.Pipeline(
```

Comment:
I disagree about this one, since we are passing so many objects around (and they have such similar names), it is necessary (at least for me) to carefully keep track of theses while we working.